htdocs/theme/eldy/dropdown.inc.php

```diff
@@ -411,7 +411,7 @@
  * QUICK ADD
  */
 #topmenu-quickadd-dropdown .dropdown-menu {
-	width: 300px !important;
+	width: 340px !important;
```

Comment:
This create problem on some smartphone. 300px was already the max to be compatible with all smartphone (some smartphone has a "logical width of 320px", and we must keep few px for space and border. Of course resolution is higher for all smartphone, but on CSS we must think logical and not real width)

Reply:
Okay, I didn't know that point. 

If we reduce to 300px, no display problem, just two columns instead of three displayed. Shall we go to 300px?

Reply:
Yes, we must keep 300px max for popup.
But we can set 340 and introduce the css
opmenu-quickadd-dropdown .dropdown-menu {
after the condition on media < 767 (mostly smartphone and tablets) so the 2 columns will occur only on smartphone and you you will 3 columns for desktop.